zebrad/tests/acceptance.rs

```diff
@@ -4,195 +4,292 @@
+
+    let child = get_child_multi_args(&["seed", "start"]);
+    let output = child.unwrap().wait_with_output()?;
+    let output = output.assert_failure()?;
 
-    let mut runner = RUNNER.clone();
-    let cmd = runner.args(&["seed", "-f"]).capture_stdout().run();
-    cmd.wait().unwrap().expect_code(1);
+    output.stdout_contains(r"unexpected free argument `start`")?;
 
-    let mut runner = RUNNER.clone();
-    let cmd = runner.args(&["seed", "start"]).capture_stdout().run();
-    cmd.wait().unwrap().expect_code(1);
+    Ok(())
 }
 
 #[test]
 fn start_no_args() -> Result<()> {
     zebra_test::init();
 
-    let (mut cmd, _guard) = test_cmd(env!("CARGO_BIN_EXE_zebrad"))?;
-
-    let mut child = cmd
-        .arg("start")
-        .stdout(Stdio::piped())
-        .stderr(Stdio::piped())
-        .spawn2()?;
+    let child = get_child_single_arg("start");
 
+    // Run the program and kill it at 1 second
     std::thread::sleep(Duration::from_secs(1));
+    let mut child_unwrapped = child.unwrap();
+    child_unwrapped.kill()?;
 
-    child.kill()?;
-
-    let output = child.wait_with_output()?;
+    let output = child_unwrapped.wait_with_output()?;
     let output = output.assert_failure()?;
 
+    // Todo: maybe add special info!() to seed command
+    // Todo: improve the regex
     output.stdout_contains(r"^(.*?)Initializing tracing endpoint")?;
 
     Ok(())
 }
 
 #[test]
-fn start_args() {
+fn start_args() -> Result<()> {
+    zebra_test::init();
+
     // Bug? This should fail but not happening
-    //let mut runner = RUNNER.clone();
-    //let cmd = runner.args(&["start", "argument"]).capture_stdout().run();
-    //cmd.wait().unwrap().expect_code(1);
+    //let child = get_child_multi_args(&["start", "argument"]);
+    //let output = child.unwrap().wait_with_output()?;
+    //let output = output.assert_failure()?;
 
-    let mut runner = RUNNER.clone();
-    let cmd = runner.args(&["start", "-f"]).capture_stdout().run();
-    cmd.wait().unwrap().expect_code(1);
+    // Invalid flag
+    let child = get_child_multi_args(&["start", "-f"]);
+    let output = child.unwrap().wait_with_output()?;
+    let output = output.assert_failure()?;
+
+    output.stdout_contains(r"unrecognized option `-f`")?;
 
     // Start + seed should be the only combination possible
-    let mut runner = RUNNER.clone();
-    let mut cmd = runner
-        .timeout(Duration::from_secs(1))
-        .args(&["start", "seed"])
-        .capture_stdout()
-        .run();
-    cmd.stdout()
-        .expect_regex(r"^(.*?)Initializing tracing endpoint");
+    let child = get_child_multi_args(&["start", "seed"]);
+
+    // Run the program and kill it at 1 second
+    std::thread::sleep(Duration::from_secs(1));
+    let mut child_unwrapped = child.unwrap();
+    child_unwrapped.kill()?;
+
+    let output = child_unwrapped.wait_with_output()?;
+    let output = output.assert_failure()?;
+
+    // Todo: maybe add special info!() to seed command
```

Comment:
I think it would be okay to add an `info` event that announces that we're starting up for `start` and `seed`

cc @hdevalence

Reply:
I think that will be good.

Reply:
I added these at https://github.com/ZcashFoundation/zebra/pull/690/commits/9d97cfb07052a44e0351904e7db87b9a0a6cef27